server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/message/ErrorBuilder.java

```diff
@@ -13,9 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kaaproject.kaa.server.operations.service.akka.messages.io.request;
+package org.kaaproject.kaa.server.transport.message;
 
+/**
+ * Converts exception into objects specific to corresponding transport channel
+ * @author Andrew Shvayka
+ *
+ */
 public interface ErrorBuilder {
 
+    /**
+     * Convert exception into objects specific to corresponding transport channel
```

Comment:
Convert the exception into objects specific to the corresponding transport channel.